redbot/cogs/mod/settings.py

```diff
@@ -370,6 +388,65 @@ async def dm(self, ctx: commands.Context, enabled: bool = None):
                 _("Bot will no longer attempt to send a DM to user before kick and ban.")
             )
 
+    @dm.command(name="banshowextrafield")
+    async def dm_banshowextrafield(self, ctx: commands.Context, enabled: bool = None):
+        """
+        Toggle whether to show an extra customizable field when banning.
+
+        This can be used to add additional information for the banned user, such as a ban appeal link.
+        """
+        guild = ctx.guild
+        if enabled is None:
+            setting = await self.config.guild(guild).ban_show_extra()
+            await ctx.send(
+                _("The extra embed field is currently set to: {setting}").format(setting=setting)
+            )
+            return
+        await self.config.guild(guild).ban_show_extra.set(enabled)
+        if enabled:
+            await ctx.send(
+                _(
+                    "An extra field will be shown when banning. Configure it with `{prefix}modset dm banextrafieldtitle` and `{prefix}modset dm banextrafieldcontents`".format(
+                        prefix=ctx.prefix
+                    )
+                )
```

Comment:
The format is currently applied before the translation.
```suggestion
                _(
                    "An extra field will be shown when banning. Configure it with `{prefix}modset dm banextrafieldtitle` and `{prefix}modset dm banextrafieldcontents`"
                ).format(prefix=ctx.prefix)
```